example/README.md

```diff
@@ -0,0 +1,36 @@
+# Example of ELK stack with Filebeat for collecting stats from LanCache
+
+This is an example set up of the ELK stack with a filebeat instance for reading the logs from LanCache. This should be a very quick way to get a working monitoring stack however should not be used in production!
+
+Please make sure to change the password of the default elastic user, this can done by searching for "changeme" and replacing. 
+
+This assumes you already have a running LanCache setup, for instructions on how to do that please see the [LanCache website](https://lancache.net/docs).
+
+## Quickstart
+
+1. Clone this repository
+2. Change the password of elasticsearch by search and replacing "changeme" in the repository.
+3. Modify the logstash.conf outputs like so
+```
+output {
+  elasticsearch {
+    hosts    => [ 'elasticsearch' ]
+    user => 'elastic'
+    password => 'changeme'
```

Comment:
Isn't this done by changine the ELSATIC_PASSWORD environment for the elasticsearch container?